addon/index.js

```diff
@@ -87,10 +86,6 @@ export class EmberChangeset extends BufferedChangeset {
   addError(key, error) {
     super.addError(key, error);
```

Comment:
https://github.com/validated-changeset/validated-changeset/blob/40676661e52ec2733ff1606607d9f68d5781a7c6/src/index.ts#L559

For example, `addError` will set the `ERRORS` object and since it is tracked, anything consuming it will pull in a new value